src/interfaces.js

```diff
@@ -170,3 +170,21 @@ interface ConfiguresRouter {
   */
   configureRouter(config: RouterConfiguration, router: Router): Promise<void>|PromiseLike<void>|void;
 }
+
+/**
+* An optional interface describing the available activation strategies.
+*/
+interface ActivationStrategy {
+  /**
+  * Reuse the existing the view model without invoking lifecycle hooks.
```

Comment:
The second `the` after existing should be removed in the comment. Same applies for the next method description